website/docs/r/lambda_function.html.markdown

```diff
@@ -52,9 +58,13 @@ resource "aws_lambda_function" "test_lambda" {
   handler       = "index.test"
 
   # The filebase64sha256() function is available in Terraform 0.11.12 and later
```

Comment:
Hi @KyMidd! 👋 Thanks for the contribution! I think this is a reasonable change, but I think the extra information within the example comments makes it a bit confusing to read. I'm thinking it might make more sense to drop this qualifier comments completely from the example, and make sure that the attribute description captures the various ways to do this instead. The example can then just capture how to do it with the archive_file data source. WDYT?

Reply:
Hey @breathingdust! 👋 . Thanks for the welcome :) That sounds reasonable to me, that example information is starting to be too much. I pushed a commit to make the changes you requested, how does that look? I'll re-request your review also.

Reply:
Hey @breathingdust, I rebased to fix a merge conflict and all tests passing. Bump for this fix to be included :)